### PR TITLE
Redirect to overview when updating Template in fromFile

### PR DESCRIPTION
**Pull request by jhadvig**
This is fixes the case when user updates the Template in the fromFile page and he is redirected to the `fromTemplate` page. By setting the template process option to `false` user will be redirected to the overview page.

@jwforres PTAL

### CODE REVIEW

File: app/scripts/directives/fromFile.js [hunk 1]
```diff
@@ -157,6 +157,10 @@ angular.module("openshiftConsole")
               openTemplateProcessModal();
             // Else if any resources already exist
             } else if (!_.isEmpty($scope.updateResources)) {
+              // If the updated resource is a Template, set the process option to false
+              if ($scope.updateResources.length === 1 && $scope.updateResources[0].kind === "Template") {
+                $scope.templateOptions.process = false;
```
Comment (jwforres) [inline app/scripts/directives/fromFile.js]: this doesn't seem right.  if I checked both the Save and Process options then it should ask me to confirm the update and then go to the from template page

Comment (jhadvig) [inline app/scripts/directives/fromFile.js]: When you are updating an existing Template we dont show the dialog in which user can pick if he wants to process or save(or both) the Template, only the one in which user has to confirm that the new one will replace the old one. After he hits `Replace` he should be redirected to the overview and not to the fromTemplate page as it is now.

Comment (jhadvig) [inline app/scripts/directives/fromFile.js]: @jwforres thoughts ?

Comment (jwforres) [inline app/scripts/directives/fromFile.js]: I think someone might get confused by that behavior. If we give you the option to process a template when you first upload it, it seems like you should get the option on an update too.

Comment (jwforres) [inline app/scripts/directives/fromFile.js]: @spadgett thoughts?

Comment (spadgett) [inline app/scripts/directives/fromFile.js]: I think the best experience would be to show the same dialog, but ask if they want to update the template or process it (instead of create and process). At least if there aren't other items in the YAML. @jhadvig how hard would that be?

**Comment by jhadvig** [comment]
@jwforres @spadgett I've update the PR based on the your comments, so now when user provides a template which already exists he will get a dialog similar to the save/process one, but instead of save he will have update. Here is the screen:
screenshot-13

PTAL

**Comment by jwforres** [comment]
I'd probably make the help text something like "This will overwrite the current version of the template."

**Comment by jhadvig** [comment]
@jwforres comment adressed :)

**Comment by jwforres** [comment]
@jhadvig you need to rebase

**Comment by jhadvig** [comment]
@jwforres rebased

File: app/scripts/directives/fromFile.js [hunk 2]
```diff
@@ -249,7 +255,7 @@ angular.module("openshiftConsole")
           var path;
           if ($scope.resourceKind === "Template" && $scope.templateOptions.process && !$scope.errorOccured) {
             var namespace = ($scope.templateOptions.add || $scope.updateResources.length > 0) ? $scope.projectName : "";
-            path = Navigate.createFromTemplateURL(resource, $scope.projectName, {namespace: namespace});
+            path = Navigate.createFromTemplateURL($scope.resourceList[0], $scope.projectName, {namespace: namespace});
```
Comment (jwforres) [inline app/scripts/directives/fromFile.js]: is this change necessary or did this get messed up in the merge conflict / rebase.  robb had just fixed an issue related to this call

Comment (jhadvig) [inline app/scripts/directives/fromFile.js]: yep. sorry for that.. didnt noticed there was this change

**Comment by jhadvig** [comment]
@jwforres updated

**Comment by jwforres** [comment]
[merge]

**Comment by jwforres** [comment]
@jhadvig was there a bug/issue associated with this fix
